app/lib/ability.rb

```diff
@@ -189,6 +188,9 @@ def initialize(user)
     end
 
     if user.site_coordinator?
+      can :manage, User, id: User.where(role: SiteCoordinatorRole.assignable_to_sites(user.role.sites)).pluck(:id)
```

Comment:
I would just check with product that we want to include all these actions, for example I'm not sure if we want to allow them to destroy users (we use suspend/unsuspend instead) so something like might be more accurate: 
```suggestion
      can [:update, :create, :update, :unsuspend, :suspend (and any other custom actions)], User, id: User.where(role: SiteCoordinatorRole.assignable_to_sites(user.role.sites)).pluck(:id)
```